digital-deck-ui/src/components/MainMenu.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div>
+    <div v-if="!isLoaded" class="main-menu">
+      <h1>Digital Deck</h1>
+      <br />
+      <span>Decks</span><br />
+      <input v-model.number="deckSelected" type="number" /><br />
+      <span>Cards on hand</span><br />
+      <input v-model.number="cardsPerPlayer" type="number" /><br />
+      <span>Cards on table</span><br />
+      <input v-model.number="cardsOnTable" type="number" /><br /><br />
+      <button @click="submitForm" style="cursor: pointer;">Start</button>
+      <p v-for="error of v$.$errors" :key="error.$uid">
+        <strong>{{ error.$message }}</strong>
+        <span v-if="error.$property === 'cardsPerPlayer'"
+          ><strong> for cards on hand</strong></span
+        >
+        <span v-else-if="error.$property === 'cardsOnTable'"
+          ><strong> for cards on table</strong></span
+        >
+        <span v-else><strong> for number of decks</strong></span>
+      </p>
+    </div>
+    <game v-if="isLoaded" />
+    <button v-if="isLoaded" class="exit-button" @click="closeSession">X</button>
+  </div>
+</template>
+
+<script>
+import Game from "./Game.vue";
+import useVuelidate from "@vuelidate/core";
+import { between, requiredUnless } from "@vuelidate/validators";
+import { VueCookies } from "vue-cookies";
```

Comment:
I don't think this is being used anywhere.

Reply:
I am not sure I will check before removing it, might be used for $cookies

Reply:
Yes, the import is needed for cookies